src/1_utils.js

```diff
@@ -263,3 +263,20 @@ utils.base64encode = function(input) {
 	}
 	return output;
 };
+
+/**
+ * Extract the deeplink path from urls in the forms of:
+ * - "protocol://domain.name/deeplink.path
+ * - "domain.name/deeplink.path"
```

Comment:
oh sorry I misunderstood your comment, these two lines are the "accepted" forms of input. Yeah, I should add output form as well. Good point.